odl/contrib/datasets/images/cambridge.py

```diff
@@ -0,0 +1,158 @@
+# Copyright 2014-2017 The ODL contributors
+#
+# This file is part of ODL.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+
+"""Images provided by the University of Cambridge."""
+
+import numpy as np
+from odl.contrib.datasets.util import get_data
+from scipy.misc import imresize
+
+__all__ = ('brain_phantom', 'resolution_phantom', 'building', 'rings',
+           'blurring_kernel')
+
+
+DATA_SUBSET = 'images_cambridge'
+URL_CAM = 'http://store.maths.cam.ac.uk/DAMTP/me404/data_sets/'
+
+
+def convert(image, shape, gray=False, dtype='float64', normalize='max'):
+    """Convert image to standardized format.
+
+    Several properties of the input image may be changed including the shape,
+    data type and maximal value of the image. In addition, this function may
+    convert the image into an ODL object and/or a gray scale image.
+    """
+
+    image.astype(dtype)
+
+    if gray:
+        image[..., 0] *= 0.2126
+        image[..., 1] *= 0.7152
+        image[..., 2] *= 0.0722
+        image = np.sum(image, axis=2)
+
+    if shape is not None:
+        image = imresize(image, shape)
+
+    if normalize == 'max':
+        image /= image.max()
+
```

Comment:
remove blankline here, makes some (including me) think that the if statement is over